### ISSUE TITLE
[Idea] Better support for touch screen devices

### ISSUE THREAD

**Issue by CloneWith** [petitlapin/Li-Ri#13]
### Issues

- Cannot input player names in the leaderboard.
- Some screens don't have Back or Menu buttons, thus making it difficult to escape and navigate.
- In gameplay, the direction changing method is confusing. I don't know how the direction changes when touching the screen, and it takes time to loop among possible directions.

### Ideas

- Add IME support or a builtin soft keyboard.
- Use Android's **back** key to go back to the previous screen or exit the game.
- Add a soft keyboard in game, which consists of four direction keys and a pause button.

**Comment by petitlapin** [comment]
Thank you for the bug and ideas!
The first version was "only" porting as it was the game on android (the only graphical change I did was to add the click on the train on top right during the game to display the menu).

* I will add the back button support as you say, that's its role.
* I would prefer not to add any text to the game (as it's an old game I've ported, the handling of assets/translations is quite different of what we use to do actually, I have to take a look on how to improve it)
* I have to check the logic for the changing of direction. For now, I kept the original one but adding the usage of arrow (in relative or absolute direction) would be nice.
